README.md

```diff
@@ -15,12 +15,14 @@ When I'm looking through the documentation of an open-source software project, I
 
 ## Setup
 
-You need to have Git, Node.js, and NPM installed.
+You need to have Git, [Node.js](https://nodejs.org/en/download/), and [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed.
```

Comment:
Doesn't NPM come with Node by default? IMHO it's redundant at best to mention it here, and confusing at worst.

Reply:
Thats a fair point, I have removed those changes in the latest commit.